docker/jdk10/x86_64/ubuntu/Dockerfile

```diff
@@ -52,11 +54,14 @@ RUN apt-get update \
 RUN mkdir -p /openjdk/target
 
 ADD sbin /openjdk/sbin
+ADD workspace/config /openjdk/config
```

Comment:
I think that we should actually be using `COPY` rather than all of these `ADD` commands. `ADD` is generally for URL's and tarballs

Reply:
Agreed, I'll do a sweep where it's appropriate